lib/utils/eol-conversion.ts

```diff
@@ -110,7 +111,9 @@ export async function readFileWithEolConversion(
 		);
 		// And summary warning later
 		globalLogger.deferredLog(
-			'Windows-format line endings were detected in some files. Consider using the `--convert-eol` option.',
+			isV12()
+				? 'Windows-format line endings were detected in some files, but were not converted due to `--no-convert-eol` option.'
```

Comment:
I didn't dig in the code to check, but could eol conversion have been skipped because the file is too large, instead of because `--no-convert-eol` was provided?

Reply:
Conversion may be skipped if too big, but this message wouldn't be displayed in that case.